### ISSUE TITLE
Settings aren't native

### ISSUE THREAD

**Issue by gregoirenovel** [warpdotdev/Warp#1880]
### Describe the bug

The Settings pane (which should btw be called "Preferences", to both align with macOS conventions as well as the "Preferences..." menu item) is not a native macOS settings pane.

### To Reproduce

Hit <kbd>cmd</kbd>+<kbd>,</kbd>

### Screenshots

![CleanShot 2022-09-26 at 14 45 34@2x](https://user-images.githubusercontent.com/1333173/192279908-260c1a84-567d-41af-ac49-84ce4a831e87.png)

### Operating System

MacOS

### OS Version

11.6.8

### Shell Version

zsh 5.8 (x86_64-apple-darwin20.0)

### Warp Version

v0.2022.09.20.08.08.stable_01

### Does this block you from using Warp daily?

No

**Comment by elviskahoro** [comment]
I think we have another open issue for this. Will try to merge this into that one.

**Comment by tonyarnold** [comment]
Confusingly for every user of macOS, Ventura changes the default to "Settings…", and any macOS prior to Ventura continues to use "Preferences…" - you get this out of the box with Apple's UI frameworks.

**Comment by skambashi** [comment]
Hi @gregoirenovel, I just wanted to check if this is still an issue. If I’m understanding correctly, you're requesting that `cmd` + `,` open the settings terminal (which it now currently does). Is that right?

**Comment by gregoirenovel** [comment]
No, my request is for your settings window to be designed according to macOS design guidelines, and not custom design conventions
